packages/bentley/grouping-mapping-widget/package.json

```diff
@@ -0,0 +1,111 @@
+{
+  "name": "@bentley/grouping-mapping-widget",
+  "version": "0.0.15",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the iTwin Reporting Platform.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "Reporting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/imodeljs/viewer-components-react/tree/master/packages/bentley/grouping-mapping-widget"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "main": "lib/main.bundle.js",
+  "types": "lib/grouping-mapping-widget.d.ts",
+  "scripts": {
+    "build": "webpack && npm run copy-files ",
+    "clean": "rimraf lib .rush/temp/package-deps*.json",
+    "copy-files": "ncp README.md lib/README.md && ncp LICENSE.md lib/LICENSE.md",
+    "cover": "",
+    "lint": "npm run lint:ts",
+    "lint:fix": "npm run lint:ts -- --fix",
+    "lint:ts": "eslint -f visualstudio \"./src/**/*.{js,ts,tsx}\" --max-warnings=0",
+    "rebuild": "npm run build",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "test": "",
+    "test:watch": "npm -s test -- --reporter min --watch-extensions ts,tsx --watch"
+  },
+  "dependencies": {
+    "@bentley/bentleyjs-core": "^2.19.25",
```

Comment:
For a pkg, you'll want most of these packages to be peerDepencies due to the coupling of static variables that they use. Otherwise you run the risk of having duplicates in your node_modules which causes all sorts of fun errors.

We tried to fix that up for 3.0, and we may some good progress, but there will continue to be some.

Reply:
most of the itwinjs core deps that is

Reply:
So the webpack plugin that avoided this has been completely deprecated then.